src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData-CSharp.Web/Program.cs

```diff
@@ -10,12 +10,13 @@
 #endif
 #if (IsOllama)
 using OllamaSharp;
-#elif (IsOpenAI || IsGHModels)
-using OpenAI;
-using System.ClientModel;
 #else
-using Azure.AI.OpenAI;
+using OpenAI;
 using System.ClientModel;
+#if (UseManagedIdentity && IsAzureOpenAI)
```

Comment:
This is causing a null reference exception creating projects when `--managed-identity` is not specified, since `UseManagedIdentity` isn't a computed property with a forced Boolean value. When checked on its own, it works, but when used with a boolean logical operator, it throws.

Our inconsistency between `Use*` and `Is*` for the computed properties has tripped us up a few times. I'm going to push a commit to this PR that standardizes on `Is*` prefixes for computed booleans.